### PR TITLE
added glove-embeddings notebook

### PR DESCRIPTION
**Pull request by jwyting**

### CODE REVIEW

File: notebooks/glove.ipynb [hunk 1]
```diff
@@ -0,0 +1,1237 @@
+{
```
Comment (Wilsven) [inline notebooks/glove.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    def pairwise_similarity(matrix_of_vectors):
</pre><p>Is there a reason why you're not directly using the <code>cosine_similarity</code> implementation in <code>sklearn.metric.pairwise</code> instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Wilsven/healthhub-content-optimization/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Comment (Wilsven) [inline notebooks/glove.ipynb:1]: <p>Is there a reason why you're not directly using the&nbsp;<code>cosine_similarity</code>&nbsp;implementation in&nbsp;<code>sklearn.metric.pairwise</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Wilsven/healthhub-content-optimization/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Comment (Wilsven) [inline notebooks/glove.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    def pairwise_similarity(matrix_of_vectors):
</pre><p>Is there a reason why you're not directly using the&nbsp;<code>cosine_similarity</code>&nbsp;implementation in&nbsp;<code>sklearn.metric.pairwise</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Wilsven/healthhub-content-optimization/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Comment (jwyting) [reply notebooks/glove.ipynb:1]: was having issues previously implementing sklearn.metric.pairwise.cosine_similarity
managed to resolve the issue. will push the revised code.
